ethers-contract/ethers-contract-abigen/src/source/online.rs

```diff
@@ -56,9 +53,9 @@ impl Explorer {
     pub const fn network(&self) -> Network {
         match self {
             Self::Etherscan => Network::Mainnet,
-            Self::Bscscan => Network::BinanceSmartChain,
-            Self::Polygonscan => Network::Polygon,
-            Self::Snowtrace => Network::Avalanche,
+            Self::Bscscan => Network::Devin,
+            Self::Polygonscan => Network::Devin,
+            Self::Snowtrace => Network::Devin,
```

Comment:
All of the block explorers like Bscscan polyscan and so on should be deleted from the Network enum. There should only be mainnet and devin.

Reply:
I will remove all unused blockchain explorers and add our Blockindex in next PR if it is okay